docs/fundamentals/networking/sockets/tcp.md

```diff
@@ -1,22 +1,35 @@
 ---
-title: Use TcpClient and TcpListener
-description: Learn how the TcpClient and TcpListener classes handle TCP communication in .NET.
+title: Use TcpClient and TcpListener in .NET
+description: Learn how to use the TcpClient class to create a socket to request and receive data using TCP in .NET.
 author: IEvangelist
 ms.author: dapine
 ms.date: 08/24/2022
-helpviewer_keywords:
-  - "requesting data from Internet, TCP"
-  - "receiving data, TCP"
-  - "TcpClient class, about TcpClient class"
-  - "data requests, TCP"
-  - "application protocols, TCP"
+helpviewer_keywords: 
+  - "protocols, TCP"
   - "network resources, TCP"
   - "sending data, TCP"
   - "TCP"
-  - "protocols, TCP"
+  - "TcpClient class, about TcpClient class"
+  - "application protocols, TCP"
+  - "receiving data, TCP"
+  - "TcpListener class, about TcpListener class"
+  - "Socket class, about Socket class"
+  - "data requests, TCP"
+  - "requesting data from Internet, TCP"
   - "Internet, TCP"
 ---
 
+# TCP Overview
+
+To use the Transmission Control Protocol (TCP) services in .NET, use the <xref:System.Net.Sockets.TcpClient> and <xref:System.Net.Sockets.TcpListener> classes. These protocol classes are built on top of the <xref:System.Net.Sockets.Socket?displayProperty=nameWithType> class and take care of the details of transferring data.
+
+The protocol classes use the underlying `Socket` class to provide simple access to network services without the overhead of maintaining state information or knowing the details of setting up protocol-specific sockets. To use asynchronous `Socket` methods, you can use the asynchronous methods supplied by the <xref:System.Net.Sockets.NetworkStream> class. To access features of the `Socket` class not exposed by the protocol classes, you must use the `Socket` class.
+
+`TcpClient` and `TcpListener` represent the network using the `NetworkStream` class. You use the <xref:System.Net.Sockets.TcpClient.GetStream%2A> method to return the network stream, and then call the stream's <xref:System.Net.Sockets.NetworkStream.ReadAsync%2A?displayProperty=nameWithType> and <xref:System.Net.Sockets.NetworkStream.WriteAsync%2A?displayProperty=nameWithType> methods. The `NetworkStream` does not own the protocol classes' underlying socket, so closing it does not affect the socket.
+
+> [!IMPORTANT]
+> <xref:System.Net.Sockets.Socket> class is highly recommended for advanced users, instead of TcpClient and TcpListener.
```

Comment:
This shouldn't be at the end. Instead of presenting this information in a highlighted section, I would consider making this an organic part of the intro paragraph (see my previous comment).

Reply:
Yeah, I agree with your previous comment, but I think having a highlighted section would be nice for this. I can move it to start of the page. Thoughts?